profiler_windows.go

```diff
@@ -0,0 +1,24 @@
+package sentry
```

Comment:
Doesn't this file also need a (windows-specific) build tag?

Reply:
> https://pkg.go.dev/go/build#hdr-Build_Constraints
> 
> Build constraints may also be part of a file's name (for example, source_windows.go will only be included if the target operating system is windows).

Reply:
Cool, TIL 👍